Firebase/Auth/Source/FIRAuth.m

```diff
@@ -18,6 +18,10 @@
 
 #import "FIRAuth_Internal.h"
 
+#if __has_include(<UIKit/UIKit.h>)
```

Comment:
Nice! For consistency, consider using the same #if for the UIKit usage below.

Reply:
Thanks! We should look at moving others over to it.

So I was considering it but I think those are probably better off with specific targets just in case since the APIs can differ (I'm fairly certain some things in UIKit are available on iOS but not tvOS, so that check may not be exactly correct).